FAQ.md: Language and interpunction

philipanda opened the pull request:
The PR applies suggestions for language improvements from https://github.com/TrenchBoot/TrenchBoot.github.io/pull/45

philipanda committed:
FAQ.md: Language and interpunction

Edited-by: Amanda Ferguson <af.jenks@gmail.com>
Signed-off-by: Filip Gołaś <filip.golas@3mdeb.com>

krystian-hebel commented on docs/FAQ.md:
```diff
@@ -64,45 +64,45 @@ The process for carrying out DTRM depends on your system:
-TrenchBoot utilizes its own system to carry out TXT on Intel, or SKINIT on AMD,
-which is a custom version of GRUB that has commands built in to carry out a
-TrenchBoot _Secure Launch_. Those commands are `slaunch` and `slaunch_module`.
+GRUB has commands built in to carry-out a TrenchBoot _Secure Launch_ (currently
```
```suggestion
GRUB has commands built in to carry out a TrenchBoot _Secure Launch_ (currently
```

krystian-hebel commented on docs/FAQ.md:
```diff
@@ -112,26 +112,27 @@ will be added later.
+make up TrenchBoot's _Security Engine_, an intermediate mini-OS that processes
+data gathered by that bootstrap phase. During the intermediate phase, the kernel
+and initramfs work together to measure block devices, individual files,
+SMBUS/DMI information etc. The kernel and initramfs also provide unseal based,
```
```suggestion
SMBUS/DMI information, etc. The kernel and initramfs also provide unseal based,
```

krystian-hebel commented on docs/FAQ.md:
```diff
@@ -112,26 +112,27 @@ will be added later.
-functionality secure launch (aka slaunch).
+functionality _Secure Launch_ (aka slaunch).
 
 ## 3. Why does TrenchBoot use an intermediate launcher?
```
```suggestion
## 3. What is the TrenchBoot intermediate loader?
```

krystian-hebel commented on docs/FAQ.md:
```diff
@@ -112,26 +112,27 @@ will be added later.
+does not fulfill is that the UEFI shim also serves as a trust delegation point
+that transitions from Microsoft Authority to Distribution/Installer/No
+Authority. The response why this is not of concern will be addressed in the
+next question.
```
```suggestion
intermediary is the UEFI shim loader and for measured boot, it is tboot. One of
the use cases for TrenchBoot is a drop-in replacement of tboot, hence a common
Linux-based loader that provides a rich security processing framework is used as
an intermediate stage. This allows for easy modifications to the framework in an
environment that most developers are familiar with. Other use cases may choose
to modify this stage, or drop it altogether if the security processing is done
in the final payload.

Intermediate loader tends to be the most common DRTM use case, it's how tboot
works and how Microsoft's Secure Core DRTM works. It allows for starting
existing operating systems with minimal or no changes.
```

krystian-hebel commented on docs/FAQ.md:
```diff
@@ -152,12 +153,12 @@ TrenchBoot is a framework that allows you to build a Linux kernel with a
 your system. You will need to use the build system to select the security
 engine components you desire, provide any necessary configurations, and build
-an instance of the loader. After that, you configure your system boot to launch
+an instance of the loader. Next configure your system boot to launch
```
```suggestion
an instance of the loader. Next, configure your system boot to launch
```

krystian-hebel commented on docs/FAQ.md:
```diff
@@ -152,12 +153,12 @@ TrenchBoot is a framework that allows you to build a Linux kernel with a
-are how feature requests are collected for the project. Check if there is a
-blueprint that is of your interest, if not, submit a blueprint via a pull
-request for a feature you would like to see implemented.
+The [TrenchBoot Blueprints](./blueprints/index.md) collect product feature
```
```suggestion
[TrenchBoot Blueprints](./blueprints/index.md) collect product feature
```

krystian-hebel reviewed (changes requested):
Please also modify the commit message by putting email address in `Edited-by` between `<>`, in case some tool depends on it.

philipanda committed:
docs/FAQ.md: Explain why TB uses an intermediate loader

docs/FAQ.md carry-out -> carry out
docs/FAQ.md: Change Section 3 title
docs/FAQ.md: interpunction
docs/FAQ.md: Remove "The" from TrenchBoot Blueprints ref

Co-authored-by: Krystian Hebel <krystian.hebel@3mdeb.com>
Signed-off-by: Filip Gołaś <filip.golas@3mdeb.com>
